src/pydicom/util/codify.py

```diff
@@ -187,7 +187,9 @@ def code_sequence(
         var_names = deque()
 
     def unique_name(name: str) -> str:
-        name_count = cast(deque, var_names).count(name) - 1
+        name_count = (
+            cast(deque, var_names).count(name) - 1
+        )  # type:ignore[redundant-cast]
```

Comment:
didn't you switch off these warnings?

Reply:
Yes, but I thought I'd leave that ignore there in case we turn them back on.